docs/source/maintenance/permissions.rst

```diff
@@ -506,7 +506,7 @@ can_add_peer
 
 Allows adding `peers <../core_concepts/glossary.html#peer>`__ to the network.
 
-A new peer will be a valid participant in the next `consensus <../core_concepts/glossary.html#consensus>`__ round after an agreement on `transaction <../core_concepts/glossary.html#transaction>`__ containing "addPeer" `command <../core_concepts/glossary.html#command>`__.
```

Comment:
The reference to consensus seems valid, maybe revert the change?

Reply:
Tbh, I did not change this part manually - it is an automated part of the doc. 
I believe it was lost because I'd deleted the section. I'll check this after I bring the section back.

Reply:
The thingy seems to have fixed itself